sql/mozfun/norm/get_windows_info/README.md

```diff
@@ -0,0 +1,49 @@
+# Windows Names, Versions, and Builds
+
+### Summary
+This function is primarily designed to parse the field `os_version` in table `mozdata.default_browser_agent.default_browser`. Though untested, the function should also parse any generic Microsoft Windows OS version strings.
+
+Given an input `os_version` string, the function returns the name of the operating system, the version name, the version number, and the build number corresponding to the operating system.
+
+As of November 2022, the expected valid values of `os_version` are either `x.y.z` or `w.x.y.z` where `w`, `x`, `y`, and `z` are integers.
+
+As of November 2022, the return values for Windows 10 and Windows 11 are based on [Windows 10 release information](https://learn.microsoft.com/en-us/windows/release-health/release-information) and [Windows 11 release information](https://learn.microsoft.com/en-us/windows/release-health/windows11-release-information). For 3-number version strings, the parser assumes the valid values of `z` in `x.y.z` are at most 5 digits in length. For 4-number version strings, the parser assumes the valid values of `z` in `w.x.y.z` are at most 6 digits in length. The function makes an educated effort to handle Windows Vista, Windows 7, Windows 8, and Windows 8.1 information, but does not guarantee the return values are absolutely accurate. The function assumes the presence of undocumented non-release versions of Windows 10 and Windows 11, and will return an estimated name, version number, build number but not the version name. The function does not handle other versions of Windows.
```

Comment:
I think the comment that accuracy is not guaranteed buries the lead slightly, could this be moved to the top? I think it's absolutely fine that this UDF cannot guarantee that it will be correct 100% of the time, but IMO if this is the case, that caveat should probably be the actual first line of the readme